src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -1473,18 +1476,7 @@ def elliptic_curve_supported(self, curve):
         group = self._lib.EC_GROUP_new_by_curve_name(curve_nid)
 
         if group == self._ffi.NULL:
-            errors = self._consume_errors()
-            self.openssl_assert(
-                curve_nid == self._lib.NID_undef
-                or errors[0]._lib_reason_match(
-                    self._lib.ERR_LIB_EC, self._lib.EC_R_UNKNOWN_GROUP
-                )
-                or
-                # This occurs in FIPS mode for unsupported curves on RHEL
-                errors[0]._lib_reason_match(
-                    self._lib.ERR_LIB_EC, self._lib.EC_R_NOT_A_NIST_PRIME
-                )
-            )
+            self._consume_errors()
             return False
```

Comment:
This is a supported check and ultimately it doesn't matter why group is null, so we should just return false after consuming the error stack.